analyzer/config/hcl/testdata/deny.rego

```diff
@@ -0,0 +1,7 @@
+package testdata
+
+deny[msg] {
+    rpl = input.spec[_].replicas
+	rpl > 2
```

Comment:
Looks like only this line has different indents.

```suggestion
    rpl > 2
```

Reply:
not really sure why, the indent looks the same in my editor :thinking: anyway I'll fix this as well

Reply:
looks like because i copy pasted the initial example, and the indent is different from my editor settings. fixed.